tests/Unit.php

```diff
@@ -2,16 +2,86 @@
 
 class Unit extends PHPUnit_Framework_TestCase {
 
-	public function subject($params = array(), $options = array()) {
-		$options += array(
-			'mock' => false,
-			'methods' => array(),
+	public static $classNamespace = "alfmarks\\";
+
+	private $subjectClass;
+	private $subjectParams = array();
+	private $subjectOptions;
+
+	private $baseParams = array();
+
+	private $baseOptions = array(
+		'mock' => false,
+		'methods' => array()
+	);
+
+	public function __construct( ...$class ) {
+		if (!empty($class)) {
+			// Since we took in a splat
+			foreach($class as $klass) {
+				$this->subjectClass = $klass;
+			}
+		} else {
+			parent::__construct();
```

Comment:
I guess since some other things inherit from Unit and call a constructor on those things that inherit it that since I added the __constructor to unit I had to account for other initialization with no args.